tests/functests/instancetype_test.go

```diff
@@ -2,23 +2,58 @@ package tests
 
 import (
 	"context"
+	"crypto/ed25519"
 	"fmt"
 	"time"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
+	"golang.org/x/crypto/ssh"
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/rand"
+	k8srand "k8s.io/apimachinery/pkg/util/rand"
 	"k8s.io/utils/ptr"
 	v1 "kubevirt.io/api/core/v1"
 	instancetypev1beta1 "kubevirt.io/api/instancetype/v1beta1"
+	"kubevirt.io/client-go/kubecli"
 )
 
-const vmReadyTimeout = 300 * time.Second
+const (
+	vmReadyTimeout = 300 * time.Second
+	sshPort        = 22
+
+	fedoraContainerDisk        = "quay.io/containerdisks/fedora:latest"
```

Comment:
Why do you use latest tag?

Reply:
Fedora is a relatively fast moving target, as such in my opinion it makes sense to always test the latest available release.